snippets/csharp/System.Security.Cryptography/Aes/Overview/program.cs

```diff
@@ -1,119 +1,80 @@
-﻿//<Snippet1>
+//<Snippet1>
 using System;
```

Comment:
The using System.IO should stay:

```suggestion
using System;
using System.IO;
```